test/uk/gov/hmrc/gform/services/HtmlGeneratorServiceSpec.scala

```diff
@@ -151,4 +151,29 @@ class HtmlGeneratorServiceSpec extends Spec {
 
     doc.getElementsByTag("td").last.html.equalsIgnoreCase("Hello") shouldBe true
   }
+
+  it should "return HTML excluding stirling pound and commas for a numeric text field" in {
+    val fieldValue = FieldValue(
```

Comment:
Please read first review from _gform-frontend_. 
This is another place when we could benefit from `ExampleData` however author of the spec intentionally decided to create test data by hand. By following this local convention it's better to leave it as it is however it's not according to global way we'd like to work. To discuss.